src/encoded/static/components/region_search.js

```diff
@@ -199,7 +199,7 @@ class AdvSearch extends React.Component {
         const region = id.query.region || '';
 
         if (this.state.genome === '') {
-            this.state.genome = context.assembly ? context.assembly : regionGenomes[0].value;
+            this.setState({ genome: context.assembly ? context.assembly : regionGenomes[0].value });
```

Comment:
Sorry; I should have noticed this aspect too from the first time. Do you have the eslint extension in your code editor? I’m not sure it would warn you for this specific thing or not, but it runs afoul of [rule 15.7](https://github.com/airbnb/javascript#comparison--unneeded-ternary).

`context.assembly || regionGenomes[0].value` would satisfy AirBnB.